ocs_ci/utility/azure_utils.py

```diff
@@ -13,15 +13,16 @@
 
 
 from ocs_ci.framework import config
-from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.exceptions import TimeoutExpiredError, TerrafromFileNotFoundException
 from ocs_ci.utility.utils import TimeoutSampler
 
 logger = logging.getLogger(name=__file__)
 
 
 # default location of files with necessary azure cluster details
 SERVICE_PRINCIPAL_FILEPATH = os.path.expanduser("~/.azure/osServicePrincipal.json")
-TERRRAFORM_FILENAME = "terraform.azure.auto.tfvars.json"
+TERRRAFORM_FILENAME = "terraform.platform.auto.tfvars.json"
+OLD_TERRRAFORM_FILENAME = "terraform.azure.auto.tfvars.json"
 
 
 def load_cluster_resource_group(cluster_path, terraform_filename=TERRRAFORM_FILENAME):
```

Comment:
Let's set this to None here.
```suggestion
def load_cluster_resource_group(cluster_path, terraform_filename=None):
```

Reply:
updated!